ironfish/src/rpc/routes/node/getStatus.ts

```diff
@@ -198,6 +202,7 @@ function getStatus(node: IronfishNode): GetStatusResponse {
       status: node.started ? 'started' : 'stopped',
       version: node.pkg.version,
       git: node.pkg.git,
+      nodeName: node.config.get('nodeName').toString(),
```

Comment:
Isn't this already a string?

Reply:
Yeah they both are strings, that toString() is not necessary here